cpp/models/abm/world.h

```diff
@@ -304,6 +308,24 @@ class World
         return m_rng;
     }
 
+    /**
+     * Get the world id. Is only relevant for graph abm or hybrid model.
+     * @return The world id
+     */
+    int get_id() const
+    {
+        return m_id;
+    }
+
+    /**
+     * Get activeness status of all persons in the world.
+     * @return Activeness vector
+     */
+    std::vector<bool>& get_activeness_statuses()
+    {
+        return m_activeness_statuses;
+    }
```

Comment:
I think something like `set_person_active(PersonId person_id, bool status)` would be better than exposing a private member.